src/Page/Shared.elm

```diff
@@ -0,0 +1,15 @@
+module Page.Shared exposing (GuideTeaser, Image)
+
+
+type alias Image =
+    { src : String, alt : String }
+
+
+type alias GuideTeaser =
```

Comment:
I think this should be in `src/Page/Guides/Data.elm` as it is tightly coupled to the full Guide resource.

Reply:
sadly that will cause a import loop but does make more sense as a category